### PR TITLE
produce reproducible output

### LINKED ISSUE #12: patch: reproductible build
Hi could you apply the following patch, that ease reproductible build:
```
Index: node-browserify-lite/index.js
===================================================================
--- node-browserify-lite.orig/index.js
+++ node-browserify-lite/index.js
@@ -79,6 +79,8 @@ function renderBundle(options, cb) {
 
   function render(entrySourcePath, cb) {
     var modules = Object.keys(sources);
+    // for reproductible build
+    module.sort();
     var aliases = {};
     modules.forEach(function(canonicalSourcePath, index) {
       aliases[canonicalSourcePath] = index;
```

### PR DESCRIPTION
**Pull request by rebecca-palmer**
Closes #12

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -116,7 +118,17 @@ function renderBundle(options, cb) {
...
-      out += "\n}, " + JSON.stringify(depMap[canonicalSourcePath]) + "],";
+      out += "\n}, {";
+      // for reproducibility, as JSON.stringify(depMap[canonicalSourcePath]) does not guarantee order - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify
+      var depMapKeys = Object.keys(depMap[canonicalSourcePath]);
+      depMapKeys.sort();
+      depMapKeys.forEach(function(depPath, index) {
+          if (index != 0) {
+              out += ",\n"; // separate to not have a trailing comma
+          }
+          out += "\"" + depPath + "\": " + depMap[canonicalSourcePath][depPath];
```
Comment (andrewrk) [inline index.js]: doesn't this need to do more escaping? This code is hard to review and it's not tested

Comment (rebecca-palmer) [reply index.js]: Escaping: Fixed.

Testing: I have now tried this on these Debian packages:
node-base64-js node-debug node-deep-eql node-err-code node-timers-browserify theano
(theano embeds dagre, dagre-d3, graphlib and graphlib-dot; the others are single modules.)

All except theano were bit-identical before and after the change.  (This makes sense as they have <=1 dependencies: in dependency maps with 0-1 entries there is nothing to sort.)

theano had what looks like the intended change, i.e. sorted dependency maps, and its d3viz examples (the part that uses JavaScript; it's mostly a Python package) appeared to work in Firefox.

**Comment by rebecca-palmer** [comment]
Did the above not address your concerns (e.g. was "not tested" intended to be a request for an actual unit test, rather than for manual testing?), or does GitHub not send notifications for replies to comments (am I supposed to click "Resolve conversation")?

**Comment by andrewrk** [comment]
Fixed by #14

**Pull request closed without merging**